<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SHPropertyDlg</class>
 <widget class="QWidget" name="SHPropertyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>193</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>物业</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnConnect">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnBack">
   <property name="geometry">
    <rect>
     <x>895</x>
     <y>10</y>
     <width>51</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_lbPort">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>100</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>端口号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_EditPort">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>250</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>5000</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_lbIP">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>100</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>IP地址:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_EditIP">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>250</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.11</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
